src/app/enzymeSetup.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { configure } from 'enzyme';
```

Comment:
This file is duplicated. Which of them do we want to retain?

Reply:
I've moved the testSetup file to the `src` folder.